### PR TITLE
Add Mac system support of pipelines

### PR DESCRIPTION
**Pull request by w5688414**
### PR types

+ New features
### PR changes

+ APIs
### Description

+ Add Mac system support
+ change dureader dataset to dureader dev

### CODE REVIEW

File: applications/experimental/pipelines/setup.py [hunk 1]
```diff
@@ -15,11 +15,18 @@
...
 
 long_description = "PIPELINES: An End to End Natural Language Proceessing Development Kit Based on ERNIE"
 
-with open("requirements.txt") as fin:
-    REQUIRED_PACKAGES = fin.read()
+if platform.system().lower() == 'windows':
+    pass
+elif platform.system().lower() == "darwin":
+    with open("requirements-cpu.txt") as fin:
+        REQUIRED_PACKAGES = fin.read()
```
Comment (tianxin1860) [inline applications/experimental/pipelines/setup.py:26]: 为什么 `darwin` 对应 cpu、`linux` 对应 gpu ?

File: applications/experimental/pipelines/ui/webapp_semantic_search.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
 
 # Adjust to a question that you would like users to see in the search bar when they load the UI:
 DEFAULT_QUESTION_AT_STARTUP = os.getenv("DEFAULT_QUESTION_AT_STARTUP",
-                                        "燃气热水器哪个牌子好?")
```
Comment (tianxin1860) [inline applications/experimental/pipelines/ui/webapp_semantic_search.py:31]: 默认的问题和答案看起来不匹配？

**Review by tianxin1860 — COMMENTED** [review]
Leave some comments

File: applications/experimental/pipelines/setup.py (hunk 1 above)
Comment (w5688414) [reply applications/experimental/pipelines/setup.py:26]: darwin是mac系统，linux以前默认的是gpu

File: applications/experimental/pipelines/ui/webapp_semantic_search.py (hunk 2 above)
Comment (w5688414) [reply applications/experimental/pipelines/ui/webapp_semantic_search.py:31]: Done

**Review by tianxin1860 — APPROVED** [review]
LGTM

**Pull request merged**